erpnext/docs/user/manual/en/schools/admission/program-enrollment.md

```diff
@@ -1,7 +1,12 @@
 # Program Enrollment
 
-This form allows you to enroll a student to a program. A student can be enrolled to multiple programs.  
+Program Enrollment describes an educational model where students must complete a  defined set of courses towards their academic objective in a specified sequence. Enrollment is a program driven process in which the student select the Program to enrol for in a Academic Year. 
```

Comment:
too general. keep it related to the module. 

Program Enrollment is used to enroll a **Student** into a particular program. A Program Enrollment is made when:

1. Student joins the institution
2. Student graduates to the next program (from 1st Grade to 2nd Grade)